web/src/types/index.ts

```diff
@@ -0,0 +1,21 @@
+export type UserRegistrationType = {
```

Comment:
I dont see any obvious reason to move it to a separate file unless you are going to reuse it

Reply:
this file will go larger an larger